src/main/java/org/sonarlint/intellij/issue/IssueProcessor.java

```diff
@@ -126,9 +127,10 @@ private static boolean shouldUpdateServerIssues(TriggerType trigger) {
         } else {
           toStore = new LiveIssue(issue, psiFile);
         }
-        map.get(psiFile.getVirtualFile()).add(toStore);
+        map.get(vFile).add(toStore);
       } catch (IssueMatcher.NoMatchException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1166)